packages/web/components/breadcrumb.tsx

```
@@ -26,7 +26,7 @@ export function Breadcrumb({
     while (current) {
       trail.unshift(current)
       if (current.parentId) {
-        current = notes.find((n) => n.id === current.parentId!)
+        current = notes.find((n) => n.id === current!.parentId)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)